clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```
@@ -2300,11 +2300,13 @@ protected void processCommonConsumerAction(ConsumerAction consumerAction) throws
 
         // Subscribe to local version topic.
         PubSubPosition subscribePosition;
-        if (consumerAction.getPubSubPosition() != null && isDaVinciClient) {
+        if (consumerAction.getPubSubPosition() != null && recordTransformer != null) {
```

If we change this check, can we fail loudly if someone tries to start a seeking client without a `recordTransformer`?